BuildResidentialHPXML/measure.rb

```diff
@@ -19,6 +19,7 @@
 require_relative '../HPXMLtoOpenStudio/resources/hvac'
 require_relative '../HPXMLtoOpenStudio/resources/lighting'
 require_relative '../HPXMLtoOpenStudio/resources/materials'
+require_relative '../HPXMLtoOpenStudio/resources/meta_measure'
```

Comment:
@shorowit Why does this need to be added?

Reply:
Because you're now calling the `get_argument_values` method.

Reply:
Do you know why it was working before?

Reply:
Not really. I got an error when trying to run one of the OSW's.

Reply:
An existing osw that was running successfully on ci? Hmm

Reply:
I ran it manually via `openstudio run -w foo.osw` 🤷‍♂️